products/guzzle.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Guzzle
+category: framework
+tags: php-runtime
+permalink: /guzzle
+releasePolicyLink: https://github.com/guzzle/guzzle?tab=readme-ov-file#version-guidance
+customColumns:
+-   property: supportedPHPVersions
+    position: after-release-column
+    label: PHP versions
+    description: Supported PHP versions range
+releases:
+-   releaseCycle: "7"
+    eol: false
+    supportedPHPVersions: ">=7.2.5,<8.5"
+    latest: "7.9.2"
+-   releaseCycle: "6"
+    eol: 2023-10-31
+    supportedPHPVersions: ">=5.5,<8.0"
+    latest: "6.5.8"
+-   releaseCycle: "5"
+    eol: 2019-10-31
+    supportedPHPVersions: ">=5.4,<7.4"
+    latest: "5.3.4"
+-   releaseCycle: "4"
+    eol: 2016-10-31
+    supportedPHPVersions: ">=5.4,<7.0"
+    latest: "4.2.4"
+-   releaseCycle: "3"
+    eol: 2016-10-31
+    supportedPHPVersions: ">=5.3.3,<7.0"
+    latest: "3.9.3"
+
+---
+
+> [Guzzle](https://docs.guzzlephp.org/en/stable/) is an HTTP client written in PHP.
+
+After each new major release, the previous release is supported for at least 2 years.
```

Comment:
I did not see this information on https://github.com/guzzle/guzzle?tab=readme-ov-file#version-guidance. Do you have the source ?

Reply:
It is mentioned here: https://github.com/guzzle/guzzle/security/policy